### DISCUSSION TITLE
User Template Variables Recognized but Empty

### DISCUSSION THREAD

**Discussion by T1mClark** [Budibase/budibase#2247] [Help]
Good Day,

First I would like to say I am excited about the capabilities (both present and planned) for Budibase, it is truly a very useful framework!

I searched on my issue, and didn't find another mention of it...  I am finding that when I attempt to use the template variables:
{{Current User.firstName}}  and {{Current User.lastName}}, they are recognized by the parser but always come up empty.

**Comment by mjashanks** [comment]
Hi @T1mClark . Thanks for the positive words!

Seems to work ok for me:

![image](https://user-images.githubusercontent.com/3524181/128144277-9bc35bd3-d953-4b50-bf7f-d228cf63a3b6.png)

Is you user's firstname and lastname set, if you look via the `Users` section:

![image](https://user-images.githubusercontent.com/3524181/128144545-251fd42d-17a5-4eb4-aaef-f06770876368.png)

**Reply by T1mClark** [reply]
Very odd.   I am using the Docker containers, I will give it a try with a CLI install and see if there is a difference.    Thank you for your time!
